Notebook.ipynb

```diff
@@ -351,69 +314,45 @@
    },
    "outputs": [],
    "source": [
-    "# ## YOU MUST PASTE YOUR OWN ROUTE INFORMATION HERE\n",
-    "my_route = 'http://model-server-sample-ovms-sughosh-dev.apps.sandbox-m2.ll9k.p1.openshiftapps.com'\n",
-    "sub_path = '/v1/models/smote/versions/1'\n",
-    "# ## YOU MUST PASTE YOUR OWN ROUTE INFORMATION HERE\n",
-    "\n",
+    "# ## YOU MUST PASTE YOUR INFERENCE LINK GOT AS AN OUTCOME OF THE ABOVE GUIDE LINK\n",
```

Comment:
@fOO223Fr

Reply:
Do you mean openvino or openvino ir formatted model? the answer to both different. Either case we do not need an explanation because modelmesh is using openvino (ovms) **under the hood** and it is not the purpose of this use-case to describe design choices of software components like Model Mesh.

Reply:
Not the ir formatted model. More about the OOB edpoints provided to user to check data which is something new in this workflow compare to previous one.

Reply:
Updated in Readme file as discussed